homeassistant/components/glances/sensor.py

```diff
@@ -341,10 +366,32 @@ def available(self) -> bool:
         return False
 
     @property
-    def native_value(self) -> StateType:
+    def native_value(self) -> StateType | datetime:
         """Return the state of the resources."""
         value = self.coordinator.data[self.entity_description.type]
 
-        if isinstance(value.get(self._sensor_name_prefix), dict):
-            return value[self._sensor_name_prefix][self.entity_description.key]
-        return value[self.entity_description.key]
+        if isinstance(value, dict):
+            if isinstance(value.get(self._sensor_name_prefix), dict):
+                return value[self._sensor_name_prefix][self.entity_description.key]
+            return value[self.entity_description.key]
+        return value
```

Comment:
Why is this being changed if it's overwritten in the new sensor? I get that you might need to add `| datetime` for implementing the `GlancesTimestampSensor`.

Reply:
Sorry, I don't get your point here, could you explain what you mean?

`GlancesSensor ` is the superclass of `GlancesTimestampSensor ` and thus both methods need the same return type...